test/cases/040_packages/023_wireguard/test-wireguard.yml

```diff
@@ -66,7 +66,8 @@ services:
 files:
   - path: etc/wireguard/wg0.conf
     contents: |
-      [Interface] 
+      [Interface]
+      Address = 192.168.2.1/24
```

Comment:
Yes, that probably makes sense even though there is no DNS server actually available as it shows how to plumb it in...